app/controllers/webhooks/records_controller.rb

```diff
@@ -73,12 +65,12 @@ def event
       nil
     end
 
-    private
-
     def view_exists?(relative_path, partial: false)
```

Comment:
If this is `:protected`, would it still be callable in auto-loaded helper(s)? Thinking probably not 🤔

Reply:
TODO: Refactor `:view_exists?` to `ApplicationController`